### PR TITLE
update env variable handling

### PR DESCRIPTION
**Pull request by burrbd**
Suggested change to env handling.

### CODE REVIEW

File: router.go [hunk 1]
```diff
@@ -43,13 +48,13 @@ func assembleMiddleware() {
 }
 
 func assembleRoutes(build, hash string) {
-	buildstamp = buildstamp
```
Comment (burrbd) [inline router.go]: ```go vet && golint``` picked this up. I notice the ```buildstamp``` is set here and also above in ```NewRouter()```. Not sure if this is intentional?

File: router.go [hunk 2]
```diff
@@ -43,13 +48,13 @@ func assembleMiddleware() {
...
+	buildstamp = build
 	githash = hash
 	var err error
 
 	// If bundling is enabled, read the RAML
 	// from bundle.go
-	if os.Getenv("BUNDLE_ASSETS") != "" {
+	if bundleAssets {
```
Comment (buddhamagnet) [inline router.go]: Not sure about creating variables for values that are only used once. We could just do:

```go
if os.Getenv(""BUNDLE_ASSETS") == "1"
```
...as a call to an env variable that does not exist will just return an empty string.

Comment (burrbd) [inline router.go]: That's a fair point. Following that logic, we could dump the ```serviceRegistration``` variable, no?

Comment (buddhamagnet) [inline router.go]: We could indeed.

File: service/main.go [hunk 3]
```diff
@@ -32,7 +32,7 @@ func init() {
 	flag.StringVar(&ramlFile, "ramlFile", "api.raml", "RAML file to parse")
 
 	serviceName = os.Getenv("SERVICE_NAME")
-	serviceRegistration = os.Getenv("SERVICE_REGISTRATION")
+	serviceRegistration = os.Getenv("SERVICE_REGISTRATION") == "1"
 	journal.Service = serviceName
```
Comment (burrbd) [inline service/main.go]: @buddhamagnet Not sure how the ```journal.Service``` works, but why assign a possible empty value to it? Condition to check for an empty ```serviceName``` happens below.

File: service/main.go [hunk 4]
```diff
@@ -32,17 +31,16 @@ func init() {
 	flag.StringVar(&ramlFile, "ramlFile", "api.raml", "RAML file to parse")
 
 	serviceName = os.Getenv("SERVICE_NAME")
-	serviceRegistration = os.Getenv("SERVICE_REGISTRATION")
-	journal.Service = serviceName
-
 	if serviceName == "" {
 		serviceName = filepath.Base(os.Args[0])
 	}
+	journal.Service = serviceName
```
Comment (burrbd) [inline service/main.go]: I assume that ```journal.Service``` should be set here?

**Comment by buddhamagnet** [comment]
Yeah all good, feel free to rebase and merge!

**Pull request merged**